--- mozilla-release/configure.orig	2014-03-19 10:50:38.758940687 +0000
+++ mozilla-release/configure	2014-03-19 10:52:04.239943420 +0000
@@ -15217,9 +15217,10 @@
 		nss_config_major_version=`$NSS_CONFIG $nss_config_args --version | \
 			sed 's/\([0-9]*\).\([0-9]*\).\([0-9]*\)/\1/'`
 		nss_config_minor_version=`$NSS_CONFIG $nss_config_args --version | \
-			sed 's/\([0-9]*\).\([0-9]*\).\([0-9]*\)/\2/'`
+			sed 's/\([0-9]*\).\([0-9]*\).*\([0-9]*\)/\2/'`
 		nss_config_micro_version=`$NSS_CONFIG $nss_config_args --version | \
 			sed 's/\([0-9]*\).\([0-9]*\).\([0-9]*\)/\3/'`
+		if [ -z "$nss_config_micro_version" ]; then nss_config_micro_version=0; fi
 		min_nss_major_version=`echo $min_nss_version | \
 			sed 's/\([0-9]*\).\([0-9]*\).\([0-9]*\)/\1/'`
 		min_nss_minor_version=`echo $min_nss_version | \
