--- BUILD/mozilla-release/configure.in.virtualenv	2012-10-24 18:32:27.000000000 +0400
+++ BUILD/mozilla-release/configure.in	2012-10-29 17:45:42.060800696 +0400
@@ -8838,7 +8838,7 @@ dnl Create a virtualenv where we can ins
 AC_MSG_RESULT([Creating Python virtualenv])
 rm -rf _virtualenv
 mkdir -p _virtualenv
-MACOSX_DEPLOYMENT_TARGET= PYTHONDONTWRITEBYTECODE= $PYTHON $_topsrcdir/other-licenses/virtualenv/virtualenv.py --system-site-packages ./_virtualenv
+MACOSX_DEPLOYMENT_TARGET= PYTHONDONTWRITEBYTECODE= virtualenv --system-site-packages ./_virtualenv
 case "$host_os" in
 mingw*)
     PYTHON=$MOZ_BUILD_ROOT/_virtualenv/Scripts/python.exe
