diff -Naur mozilla-1.9.2/layout/generic/nsFrame.h mozilla-1.9.2.tpg/layout/generic/nsFrame.h
--- mozilla-1.9.2/layout/generic/nsFrame.h	2009-10-29 22:10:54.000000000 +0000
+++ mozilla-1.9.2.tpg/layout/generic/nsFrame.h	2009-11-08 18:16:01.000000000 +0000
@@ -152,6 +152,8 @@
    */
   friend nsIFrame* NS_NewEmptyFrame(nsIPresShell* aShell,
                                     nsStyleContext* aContext);
+ // Make it public like it was in 1.9.1
+  void operator delete(void* aPtr, size_t sz);
 
 private:
   // Left undefined; nsFrame objects are never allocated from the heap.
@@ -168,7 +170,9 @@
   // worse, some C++ compilers will synthesize calls to this function
   // from the "deleting destructors" that they emit in case of
   // delete-expressions, so it can't even be undefined.
-  void operator delete(void* aPtr, size_t sz);
+
+  // revert to make it public, see above declaration
+ //  void operator delete(void* aPtr, size_t sz);
 
 public:
 
